### PR TITLE
Rebranded the entire application UI

### PR DESCRIPTION
**Pull request by alvarengathomas**
## Summary

Complete UI changes to follow Hemi Designs

## Related Issue

closes https://github.com/hemilabs/pure.finance/issues/39

## Screenshots (if appropriate):

Captura de Tela 2024-09-12 às 12 12 09

Captura de Tela 2024-09-12 às 12 12 49

## Types of changes

- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: site/components/Input.js [hunk 1]
```diff
@@ -26,9 +28,28 @@ const SuffixedInput = ({ suffix, ...props }) => (
...
+    }
+
+    if (captionColor.includes('green')) {
+      setName('check')
+    }
+  }, [])
+
+  return <SvgContainer className={`w-6 h-6`} name={name} />
+}
+
```
Comment (gndelia) [inline site/components/Input.js:52]: I think `useState` and `useEffect` are not needed here. You can rewrite this as just deriving values from props:

```suggestion
const CaptionIcon = ({ captionColor }) => {
  const name = captionColor.includes('red')
       ? 'error'
       : 
          captionColor.includes('green')
             ? 'check'
             : ''

  return <SvgContainer className={`w-6 h-6`} name={name} />
}
```

you can rewrite this to use i/else or whatever you prefer, but the point is that both the state and the effect are not needed.

File: site/components/layout/Breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            />
+          </div>
+        )}
+        className="text-gray-600 cursor-pointer"
+      >
+        <ul className="w-54 absolute z-10 flex flex-col mt-1 p-4 text-left bg-white rounded-xl shadow-lg">
+          {utilities.map(({ title, href, selected }) => (
+            <Link href={href} key={title}>
+              <div className="hover:bg-slate-50 flex items-center justify-between px-3 py-2 rounded-md">
+                <li className={`${selected ? 'text-black' : ''} `}>{title}</li>
```
Comment (gndelia) [inline site/components/layout/Breadcrumb.js:38]: Really minor, but as it's only one style, no need for text interpolation

```suggestion
                <li className={selected ? 'text-black' : undefined}>{title}</li>
```

File: site/components/layout/HelperBox.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        {helperText.title && (
+          <h1 className="text-2xl font-normal">{helperText.title}</h1>
+        )}
+        {helperText.text && (
+          <h4 className="text-slate-500 mt-4 text-sm">{helperText.text}</h4>
+        )}
+      </div>
+      <div className="mt-32">
+        <h4 className="text-slate-500 mb-4 pb-2 text-sm border-b">
+          Got Questions?
```
Comment (gndelia) [inline site/components/layout/HelperBox.js:21]: same here re: translations

File: site/components/layout/HomeLayout.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+const HomeLayout = ({ children, title }) => (
+  <>
+    <Head title={title} />
+    <div className={`w-full ${inter.className}`}>
+      <div className="">
+        <Navbar />
+        <Hero />
+        <div className="container">{children}</div>
+        <Footer />
+      </div>
```
Comment (gndelia) [inline site/components/layout/HomeLayout.js:17]: empty `className` can be removed. 

```suggestion
      <div>
        <Navbar />
        <Hero />
        <div className="container">{children}</div>
        <Footer />
      </div>
```

I also wonder if then, the whole div can be removed

```suggestion
        <Navbar />
        <Hero />
        <div className="container">{children}</div>
        <Footer />
```

File: site/pages/[locale]/merkle-claims.js [hunk 5]
```diff
@@ -8,10 +8,28 @@ import watchAsset from 'wallet-watch-asset'
...
+        'Merkle Claims use a Merkle tree structure to securely and efficiently verify and distribute tokens to users based on their claim IDs.'
+    },
+    {
+      title: 'How do I find my Claim ID?',
+      answer:
+        'Your Claim ID is usually provided by the issuer of the rewards. Check the communication or platform where the rewards were announced.'
+    }
+  ]
+}
+
```
Comment (gndelia) [inline site/pages/[locale]/merkle-claims.js:32]: same old here re: translations

File: site/components/Input.js (hunk 1 above)
Comment (alvarengathomas) [reply site/components/Input.js]: done

File: site/components/layout/HelperBox.js (hunk 3 above)
Comment (alvarengathomas) [reply site/components/layout/HelperBox.js]: done

File: site/components/layout/HomeLayout.js (hunk 4 above)
Comment (alvarengathomas) [reply site/components/layout/HomeLayout.js]: done

File: site/pages/[locale]/merkle-claims.js (hunk 5 above)
Comment (alvarengathomas) [reply site/pages/[locale]/merkle-claims.js]: done

File: site/components/layout/Breadcrumb.js (hunk 2 above)
Comment (alvarengathomas) [reply site/components/layout/Breadcrumb.js]: done

File: site/components/InputBalance.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+const Balance = ({ balance, showMax, setMax }) => {
+  const t = useTranslations()
+
+  return (
+    <div className="flex gap-1 items-center">
+      <span className="text-slate-500">{t('balance')}:</span>
+      <label className="text-black text-sm">{balance}</label>
+      {showMax && (
+        <label className="text-orange-950 ml-1 cursor-pointer" onClick={setMax}>
+          MAX
```
[the remark this answers is not in the document]
Comment (alvarengathomas) [reply site/components/InputBalance.js]: done

File: site/components/layout/Hero.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+const Hero = () => (
+  <div className="border-orange-950 from-orange-opacity relative flex flex-col items-center justify-center -mt-24 pb-20 px-8 bg-gradient-to-t border-b overflow-hidden">
+    <div className="relative flex justify-center">
+      <div className="w-128 absolute">
+        <HemiLogoDashed />
+      </div>
+    </div>
+    <div className="z-10 mt-64 md:mt-72">
+      <h1 className="max-w-xl text-center text-4xl font-semibold md:max-w-3xl md:text-5xl">
+        Empower Your Web3 Journey with{' '}
```
Comment (gndelia) [inline site/components/layout/Hero.js:12]: this also requires translation, as well as line 13

Comment (alvarengathomas) [reply site/components/layout/Hero.js]: done

**Review by gndelia — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]
uACK

**Pull request merged**